pkg/assembler/clients/operations/certifyBad.graphql

```diff
@@ -19,7 +19,7 @@
 
 mutation CertifyBadPkg($pkg: PkgInputSpec!, $pkgMatchType: MatchFlags, $certifyBad: CertifyBadInputSpec!) {
   ingestPackage(pkg: $pkg) {
-    ...allPkgTree
+    ...AllPkgTree
```

Comment:
hmm, any particular reason we’re changing this? probably need to change this on the examples side as well?

Reply:
to make them public so that I can use them in the unit tests. Before they were all private. Yeah I can change the examples also.

Reply:
Seems like we should do it for the rest also as they should be public.

Reply:
Sure. I didn't realize that these being package private was an issue. But there's no problem making them public for testing, except that when we decide to reduce the amount of data sent on the wire (e.g., maybe we realize we only need an ID for ingested stuff instead of everything), the test code will break.